app/web/endpoints/url.py

```diff
@@ -21,65 +21,101 @@
+                detail="User has reached their monthly URL quota.",
+            )
         if not user.has_quota_max_monthly_mbs(archive.group_id):
-            raise HTTPException(status_code=429, detail="User has reached their monthly MB quota.")
+            raise HTTPException(
+                status_code=HTTPStatus.TOO_MANY_REQUESTS,
+                detail="User has reached their monthly MB quota.",
+            )
         group_queue = user.priority_group(archive_create.group_id)
     else:
         archive_create.author_id = archive.author_id or email
         group_queue = convert_priority_to_queue_dict("high")
 
-
-    task = celery.signature("create_archive_task", args=[archive_create.model_dump_json()]).apply_async(**group_queue)
+    task = celery.signature(
+        "create_archive_task", args=[archive_create.model_dump_json()]
+    ).apply_async(**group_queue)
     task_response = schemas.Task(id=task.id)
-    return JSONResponse(task_response.model_dump(), status_code=201)
+    return JSONResponse(
+        task_response.model_dump(), status_code=HTTPStatus.CREATED
+    )
 
 
 @url_router.get("/search", summary="Search for archive entries by URL.")
 def search_by_url(
-        url: str, skip: int = 0, limit: int = 25,
-        archived_after: datetime = None, archived_before: datetime = None,
-        db: Session = Depends(get_db_dependency),
-        email: str = Depends(get_token_or_user_auth)
+    url: str,
+    skip: int = 0,
+    limit: int = 25,
+    archived_after: datetime = None,
+    archived_before: datetime = None,
+    db: Session = Depends(get_db_dependency),
+    email: str = Depends(get_token_or_user_auth),
 ) -> list[schemas.ArchiveResult]:
-
     read_groups, read_public = False, False
     if email != ALLOW_ANY_EMAIL:
         user = UserState(db, email)
         if not user.read and not user.read_public:
-            raise HTTPException(status_code=403, detail="User does not have read access.")
+            raise HTTPException(
+                status_code=HTTPStatus.FORBIDDEN,
+                detail="User does not have read access.",
+            )
         read_groups = user.read
         read_public = user.read_public
-    return crud.search_archives_by_url(db, url.strip(), email, read_groups, read_public, skip=skip, limit=limit, archived_after=archived_after, archived_before=archived_before)
+    return crud.search_archives_by_url(
+        db,
+        url.strip(),
+        email,
+        read_groups,
+        read_public,
+        skip=skip,
+        limit=limit,
+        archived_after=archived_after,
+        archived_before=archived_before,
+    )
 
 
 @url_router.delete("/{id}", summary="Delete a single URL archive by id.")
 def delete_archive(
-    id:str,
+    id: str,
     user: UserState = Depends(get_user_state),
-    db: Session = Depends(get_db_dependency)
-) -> schemas.DeleteResponse:
+    db: Session = Depends(get_db_dependency),
+) -> JSONResponse:
```

Comment:
DeleteResponse seems better for /docs interpretation

Reply:
Done.